src/components/Form/Form.tsx

```diff
@@ -1 +1,36 @@
-export {}
+import React from 'react';
+import './Form.css';
+
+
+interface userSentimentProps {
+  userSentiment: (newSentiment: number) => void
+}
+
+const Form: React.FC<userSentimentProps> = props => {
+  const assignSentiment = (event: React.MouseEvent<HTMLButtonElement>) => {
+      event.preventDefault();
+      const button: HTMLButtonElement = event.currentTarget;
+      props.userSentiment(Number(button.name));
+  }
+
+  return(
+    <div className="form-container">
+      <section className="questionnaire">
+        <h2>how are you feeling today?</h2>
+        <article className="sentiment-selection">
```

Comment:
A `<div>` might be more appropriate here in lieu of an `<article>`. MDN on `<article>`: "a forum post, a magazine or newspaper article, or a blog entry, a product card, a user-submitted comment, an interactive widget or gadget, or any other independent item of content." The `<h2>` and the `<article>`'s children are intimately related, and probably not independent items of content.

Reply:
I would consider wrapping the question in a `<form>`. Though it is only one question, it is an interactive portion where the user submits information (a sentiment). What do you think?

Reply:
I changed it to <form> although I wonder if it really is much of a form? But Anthony makes a great point about it not seeming like an article, so form certainly feels like the next best option!